src/sentry/eventstore/models.py

```diff
@@ -197,6 +201,29 @@ def location(self):
             return self._snuba_data[column]
         return super(Event, self).location
 
+    def get_search_message(self, event_metadata=None, culprit=None):
```

Comment:
Is this change actually needed for fetching data from nodestore ?

Reply:
It's a dependency. This was kind of a hack before since the message from snuba is different from the message from nodestore. We're no longer providing a way to explicitly say that you want the column to be loaded from snuba (a good thing since this was being abused before). I'm going to try to improve this a bit more in a separate PR.

Reply:
See https://github.com/getsentry/sentry/pull/16473 for a more proper fix for the event.message / event.search_message behaviour